### ISSUE TITLE
Table.Row - Context.Provider missing 'value' prop

### ISSUE THREAD

**Issue by MarosPistej** [segmentio/evergreen#1140]
**Description**
There seems to be bug in `Table.Row` that `value` prop is missing in `<Context.Provider>` as described in console warning

```js
Warning: The `value` prop is required for the `<Context.Provider>`. Did you misspell it or forget to pass it?
    at TableRow
```
![image](https://user-images.githubusercontent.com/43110161/105871560-b7cb9500-5ff9-11eb-8462-54e489380cf5.png)

**Additional context**
Code sandbox warning example using `evergreen-ui` table `sample` bellow
[Warning example](https://codesandbox.io/s/blazing-wind-4bkv6?file=/src/App.js)

**Comment by billyct** [comment]
🌝 i downgrade the react version from `17.x` to `16.x`, then the warning disappeared.

`react@17.x` Warn when no value is provided to <Context.Provider> 
https://github.com/facebook/react/pull/19054

**Comment by tomhands1** [comment]
I also have this issue, after upgrading versions of React. It appears that setting a value to context providers is now a required field and throws a warning if not as of:
- https://github.com/facebook/react/issues/19020
- https://github.com/facebook/react/pull/19054
Is there a workaround or solution that doesn't involve downgrading react versions.
Thanks

**Comment by dikaio** [comment]
Also interested in this.

**Comment by Samuel-Montoya** [comment]
Getting the same warning on my end!

**Comment by mshwery** [comment]
Resolved by #1256

**Comment by jgibson94** [comment]
> Resolved by #1256

Unfortunately i'm still getting this exact error on 6.1.0

**Comment by mshwery** [comment]
> Unfortunately i'm still getting this exact error on 6.1.0

Yep! I don't think we've cut a release since that PR got merged

**Comment by jgibson94** [comment]
> > Unfortunately i'm still getting this exact error on 6.1.0
> 
> Yep! I don't think we've cut a release since that PR got merged

My mistake!! :-)

**Comment by Sti2nd** [comment]
Fixed in 6.2.0 https://github.com/segmentio/evergreen/releases/tag/v6.2.0
